app/views/search/_form.html.erb

```diff
@@ -220,6 +211,10 @@ end
       </div>
     </details>
   </div>
+
+  <div class="basic-search-submit">
+    <button type="submit" class="btn button-primary">Search</button>
```

Comment:
I just checked our style guide for other button color options and none of them change the non-hover state. I swear Bento used to have green button links, but maybe they were local overrides. Oh well. One day we'll have a non-greyscale UI element to use.

Reply:
Yeah, I saw the same. We could implement a custom style here if it's just not clear that it's a button, but tbh I think eliminating the `border-radius: 0` override will go a long way toward making it more button-like.

Reply:
Yeah I think it's pretty clear it's a button and what it does, I also just feel like we convinced ourselves years ago that all colors are bad somehow.